PixelHunter1995/Utilities/Polygon.cs

```diff
@@ -386,20 +426,106 @@ private int EdgeAt(Vector2 vertexA, Vector2 vertexB)
             return -1;
         }
 
-        public (Vector2, double) GetClosestVertex(Vector2 position)
+        private static Vector2 ProjectOntoLine(Vector2 pointOfInterest, Vector2 pointA, Vector2 pointB)
```

Comment:
Since pointA and pointB together represent one thing, perhaps they should in fact be a tuple?

Reply:
Or maybe that's not practical, I dunno